### PR TITLE
Feature/gh 6540

### LINKED ISSUE #6540: Prepare basic views for Users
Depends on #6538
- [x] add impex
- [x] Listing page
- [x] Details page
- [x] Form
- [x] Create page
- [x] Edit page

### PR DESCRIPTION
**Pull request by mszczepaniak**
Closes #6540

### CODE REVIEW

**Review by Pucek9 — CHANGES_REQUESTED** [review]
1 disable / enable works only after refresh page
2 user form style doesnt extend forms as others (@extend %cx-organization-form;)
3 Create/Cancel should move to Users, Edit/Cancel -> Details
4 Update user doesn't work at all
5 Create user
request:
```
firstName: "test"
isAssignedToApprovers: false
lastName: "dt"
orgUnit: {uid: "Test 3"}
roles: ["b2bcustomergroup"]
titleCode: "mr"
uid: "aaa@fff.com"
```
response
```
   "errors" : [ {
      "message" : "profile.email.invalid",
      "reason" : "missing",
      "subject" : "email",
      "subjectType" : "parameter",
      "type" : "ValidationError"
   } ]
}
```
& verify behavior of assignee to disabled org unit
6. Create should redirect to details page
7. Fix translations (userGroups/users, user/approvers etc.)
8. CI

File: projects/core/src/organization/store/effects/b2b-user.effect.spec.ts [hunk 1]
```diff
@@ -21,6 +21,7 @@ import { B2BSearchConfig } from '../../model/search-config';
 import { B2BUser } from '../../../model/org-unit.model';
 import { Permission } from '../../../model/permission.model';
 import { UserGroup } from '../../../model/user-group.model';
+import { RoutingService } from '../../../../src/routing';
```
Comment (Pucek9) [inline projects/core/src/organization/store/effects/b2b-user.effect.spec.ts]: path seems strange
I can see a lot of unnecessary `/../src/` below too

File: projects/core/src/organization/store/effects/b2b-user.effect.ts [hunk 2]
```diff
@@ -14,6 +14,7 @@ import { B2BUser } from '../../../model/org-unit.model';
 import { B2BUserConnector } from '../../connectors/b2b-user/b2b-user.connector';
 import { Permission } from '../../../model/permission.model';
 import { UserGroup } from '../../../model/user-group.model';
+import { RoutingService } from '../../../../src/routing';
```
Comment (Pucek9) [inline projects/core/src/organization/store/effects/b2b-user.effect.ts]: path seems strange

File: projects/storefrontlib/src/cms-components/organization/units/index.ts [hunk 3]
```diff
@@ -12,3 +12,18 @@ export * from './unit-address-edit/unit-address-edit.module';
...
+export * from './unit-users/unit-users.component';
+export * from './unit-assign-roles/unit-assign-roles.component';
+export * from './unit-assign-approvers/unit-assign-approvers.component';
+export * from './unit-approvers/unit-approvers.component';
+export * from './unit-manage-addresses/unit-manage-addresses.component';
+export * from './unit-address-create/unit-address-create.component';
+export * from './unit-address-edit/unit-address-edit.component';
+export * from './unit-address-details/unit-address-details.component';
+export * from './unit-children/unit-children.component';
+export * from './unit-cost-centers/unit-cost-centers.component';
```
Comment (Pucek9) [inline projects/storefrontlib/src/cms-components/organization/units/index.ts]: why do you export components? we export modules for that

File: projects/storefrontlib/src/cms-components/organization/users/user-approvers/user-approvers.component.html [hunk 4]
```diff
@@ -3,21 +3,21 @@
   [links]="[
     {
       cxRoute: 'userDetails',
-      name: 'user.details',
+      name: 'userForm.details',
```
Comment (Pucek9) [inline projects/storefrontlib/src/cms-components/organization/users/user-approvers/user-approvers.component.html]: that's not form component, the same with other below

File: projects/storefrontlib/src/cms-components/organization/users/user-assign-approvers/user-assign-approvers.component.html [hunk 5]
```diff
@@ -3,21 +3,21 @@
   [links]="[
     {
       cxRoute: 'userDetails',
-      name: 'user.details',
+      name: 'userForm.details',
```
Comment (Pucek9) [inline projects/storefrontlib/src/cms-components/organization/users/user-assign-approvers/user-assign-approvers.component.html:6]: + here too

File: projects/storefrontlib/src/cms-components/organization/users/user-approvers/user-approvers.component.html (hunk 4 above)
Comment (mszczepaniak) [reply projects/storefrontlib/src/cms-components/organization/users/user-approvers/user-approvers.component.html]: The name property is used by fake tabs component for translations only, and one could argue that tab elements are form elements. User translations should only contain user properties.

File: projects/core/src/organization/store/effects/b2b-user.effect.spec.ts (hunk 1 above)
Comment (mszczepaniak) [reply projects/core/src/organization/store/effects/b2b-user.effect.spec.ts]: Weird vscode autoimports fixes, fixed.

File: projects/core/src/organization/store/effects/b2b-user.effect.ts (hunk 2 above)
Comment (mszczepaniak) [reply projects/core/src/organization/store/effects/b2b-user.effect.ts]: Weird vscode autoimports fixes, fixed.

File: projects/storefrontlib/src/cms-components/organization/units/index.ts (hunk 3 above)
Comment (mszczepaniak) [reply projects/storefrontlib/src/cms-components/organization/units/index.ts]: Was just an experiment to tackle the prod storefrontlib build issue, already solved and brought back.

**Review by Pucek9 — APPROVED** [review]

**Pull request merged**